src/app/accessRequests/views/reviewOrganisationRequest.ejs

```diff
@@ -96,7 +96,13 @@
 
                     </fieldset>
                 </div>
-
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
```

Comment:
to make the text regular, this can be replaced with the following 
`<p class="govuk-warning-text__text govuk-!-font-weight-regular">`

Reply:
done to design principle